web/struct_tree_layer_builder.js

```diff
@@ -104,7 +114,24 @@ class StructTreeLayerBuilder {
   #setAttributes(structElement, htmlElement) {
     const { alt, id, lang } = structElement;
     if (alt !== undefined) {
-      htmlElement.setAttribute("aria-label", removeNullCharacters(alt));
+      // Don't add the label in the struct tree layer but on the annotation
+      // in the annotation layer.
+      let added = false;
+      const label = removeNullCharacters(alt);
+      for (const child of structElement.children) {
+        if (child.type === "annotation") {
+          let attrs = this.#elementAttributes.get(child.id);
+          if (!attrs) {
+            attrs = Object.create(null);
```

Comment:
Can't we make this a `Map` instead, so that it can be iterated "directly" without having to use `Object.entries(...)` in the annotationLayer-code above?
https://github.com/mozilla/pdf.js/pull/18658/files#diff-d7b040092cf6be09a276b785ba03c46e79bc8342cf4ae4fb1bc139dea8513425R3121